route/dht.go

```diff
@@ -0,0 +1,17 @@
+package route
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/thunderdb/ThunderDB/rpc"
+	"net"
+)
+
+func InitDhtServer(l net.Listener) (server *rpc.Server, err error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function InitDhtServer should have comment or be unexported